src/AvahiMDNSManager.cpp

```diff
@@ -861,6 +861,17 @@ void MDNSManager::setErrorHandler(MDNSManager::ErrorHandler handler)
     pimpl_->errorHandler = handler;
 }
 
+void MDNSManager::registerAddress(MDNSService &service,
+                                  ErrorCodeHandler async_result)
+{
+    // TODO
```

Comment:
throw exception so it doesn't fail silently?

Reply:
ok